src/foam/core/Null.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.core',
+  name: 'Null',
+  documentation: "An axiom which adds null implementations for methods of 'of'.",
```

Comment:
`Proxy` extends Property because it also holds a delegate. Since `Null` doesn't need to store anything I decided not to extend Property here.